packages/next/next-server/server/config.ts

```diff
@@ -228,6 +228,14 @@ export default function loadConfig(
       phase,
       userConfigModule.default || userConfigModule
     )
+
+    if (Object.keys(userConfig).length === 0) {
+      console.warn(
+        chalk.yellow.bold('Warning: ') +
+          'Detected next.config.js, no exported configuration found.'
```

Comment:
The message should have a err.sh link like the other error messages. You can add this by adding a markdown file to the `errors` directory in the root of the repository and linking to it like this: `https://err.sh/zeit/next.js/<name_of_the_file_with_dot_md>`

Reply:
Will do 👍

Reply:
Added https://github.com/zeit/next.js/pull/10228/commits/568286d34b6c572b1cefbec798e1e0951c0f7a51